src/cmd/sanity_test/export/groups.go

```diff
@@ -50,6 +50,20 @@ func CheckGroupsExport(
 		envs)
 }
 
+func removeDeletedMessages(
```

Comment:
@ryanfkeepers I found us to be not exporting deleted messages, just wanted to make sure this is the intended behavior.

Reply:
@ryanfkeepers could you please review this PR? I don't have enough context on the sanity tests